src/ui/pages/error-page.svelte

```diff
@@ -105,25 +115,50 @@
   }
 </script>
 
-<MessageLayout
-  title={getTranslatedErrorTitle()}
-  {onDismiss}
-  type="error"
-  closeButtonTitle={$translator.translate(LocalizationKeys.ErrorButtonTryAgain)}
->
-  {#snippet icon()}
-    <IconError />
-  {/snippet}
+{#if error.errorCode === PurchaseFlowErrorCode.AlreadyPurchasedError}
+  <MessageLayout
+    title={getTranslatedErrorTitle()}
+    {onDismiss}
+    type="error"
+    closeButtonTitle={$translator.translate(
+      LocalizationKeys.ErrorPageCloseButtonTitle,
+      { appName: appName ?? "App" },
```

Comment:
Checking if this button actually works as would be expected. Default string in messageLayout being "Go back to app" implies so, but validating 👀

Reply:
Thanks for confirming. Yeah it seems fine in the example app at least I've been using that throughout, but I can only take that so far. 
An e2e test is a good point, I don't think any of the existing ones go this far through - I'll give it a try.

Reply:
@nicfix I've added a small test https://github.com/RevenueCat/purchases-js/pull/478/commits/a799c3e65132b3425276ec34c6641d3f1e94c861, just need to r[esolve the (unrelated I think) CI errors](https://revenuecat.slack.com/archives/C03UKCQLW1Z/p1747310354783819) and this should be good to merge.